spring-boot-project/spring-boot-actuator/src/main/java/org/springframework/boot/actuate/metrics/web/jetty/JettyConnectionMetricsBinder.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2012-2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.boot.actuate.metrics.web.jetty;
+
+import java.util.Collections;
+
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.Tag;
+import io.micrometer.core.instrument.binder.jetty.JettyConnectionMetrics;
+import org.eclipse.jetty.server.Server;
+
+import org.springframework.boot.context.event.ApplicationStartedEvent;
+import org.springframework.boot.web.context.WebServerApplicationContext;
+import org.springframework.boot.web.embedded.jetty.JettyWebServer;
+import org.springframework.boot.web.server.WebServer;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.ApplicationListener;
+
+/**
+ * Binds {@link JettyConnectionMetrics} in response to the
+ * {@link ApplicationStartedEvent}.
+ *
+ * @author Chris Bono
+ * @since 2.6.0
+ */
+public class JettyConnectionMetricsBinder implements ApplicationListener<ApplicationStartedEvent> {
+
+	private final MeterRegistry meterRegistry;
+
+	private final Iterable<Tag> tags;
+
+	public JettyConnectionMetricsBinder(MeterRegistry meterRegistry) {
+		this(meterRegistry, Collections.emptyList());
```

Comment:
@wilkinsona I followed suit w/ what was done in the sibling `JettyServerThreadPoolMetricsBinder` and chained no-arg `<init> -> <init w/ tags>`.  This makes sense as custom binders may want this. 

Feels like we should add sanity coverage around this case. I am happy to do that for all 3 in this proposal. Or was it intentionally left out of the coverage?